bazel/grpc_build_system.bzl

```diff
@@ -168,7 +168,7 @@ def ios_cc_test(
             deps = ios_test_deps,
         )
 
-def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++", size = "medium", timeout = None, tags = [], exec_compatible_with = [], exec_properties = {}, shard_count = None, flaky = None):
+def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++", size = "medium", timeout = None, tags = [], exec_compatible_with = [], exec_properties = {}, shard_count = None, flaky = None, rbe_linux_only_uses_blackhole_ipv6_address = False):
```

Comment:
btw, IF we accept this change, we would need to add the new argument in google3 also, otherwise the import would fail.

Reply:
I didn't plan on porting this black hole mechanism to google3 (I'm pretty sure it's not going to be possible, because in test binaries need to be running with the NET_ADMIN capability in order for it to work - I believe this is only going to be possible in linux RBE).

Reply:
It's fine to not port to google3, but you'd still need to add a new dummy parameter `rbe_linux_only_uses_blackhole_ipv6_address = False` to the internal definition of grpc_cc_test() otherwise the import will fail (try cherrypicking into internal and you'll see).

But the best option would be not having the new arg `rbe_linux_only_uses_blackhole_ipv6_address = False` at all.
Ideally you'd be able to just use a custom sh_test and tweak the exec_properies just for that one test.